src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,224 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { PROFILE_CARD_SPACING, PROFILE_CARD_TYPOGRAPHY } from '../../style-library/theme/theme';
+import { VARIANT_CONFIGS } from './ProfileCard.constants';
+
+import type {
+  StyledProfileCardProps,
+  ProfileCardSize,
+  ProfileCardVariant,
+} from './ProfileCard.types';
+
+export const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size }) => {
+  const spacingConfig = PROFILE_CARD_SPACING[$size];
+
+  return {
+    padding: theme.spacing(spacingConfig.paddingUnits),
+    borderRadius: theme.spacing(2),
+    boxShadow: theme.shadows[2],
+    transition: theme.transitions.create(['box-shadow', 'transform'], {
+      duration: theme.transitions.duration.short,
+    }),
+    '&:hover': {
+      boxShadow: theme.shadows[4],
+      transform: 'translateY(-2px)',
```

Comment:
The hover transform uses a hardcoded pixel value (`translateY(-2px)`). The project styling rules discourage hardcoded px values and recommend using the theme spacing system.

**Sugerencia:**
```typescript
Use `transform: `translateY(-${theme.spacing(0.25)})`` or compute using `theme.spacing()` to avoid hardcoded pixels and keep consistent scaling with the design system.
```